app/views/passengers/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<h2>Passenger Details</h2>
+
+<div class="intro">
+  <div><strong>Name:</strong> <%= @passenger.name %></div>
+  <div><strong>Phone:</strong> <%= @passenger.phone_num %></div>
+
+  <strong><%= "#{@passenger.name} has spent a total of #{@passenger.total_cost}." %></strong>
+</div>
+
+<div class="button">
+<%= link_to "Request Trip", passenger_trips_path(@passenger.id), method: :post, data: {confirm: "Do you really want to request a trip for #{@passenger.name}?"} %>
+</div>
+
+<table class="passengers_show">
```

Comment:
This is a nitpick, but for CSS class names, we use the `-` rather than the `_` to separate words, so this would be `passengers-show` instead.